README.md

```diff
@@ -4,12 +4,53 @@
 [![Coverage report](https://gitlab.laas.fr/loco-3d/sl1m/badges/master/coverage.svg?job=doc-coverage)](http://projects.laas.fr/gepetto/doc/loco-3d/sl1m/master/coverage/)
 
 
-Python dependencies:
-	scipy
-	quadprog
-	matplotlib
-	pycddlib
-	cvxpy
+## Python dependencies:
+	
+	python -m pip install scipy
+	sudo apt install robotpkg-py38-quadprog
+	python -m pip install matplotlib
+	python -m pip install pycddlib
+	python -m pip install cvxpy
 
 :warning: this package requires gurobi :warning:
 A free academic license can be obtained at https://www.gurobi.com
+
+## Demo dependencies:
+
+These are the dependencies for the robot and rbprm server.
+```
+sudo apt install -y robotpkg-py38-hpp-rbprm \
+					robotpkg-py38-hpp-rbprm-corba \
+					robotpkg-py38-anymal-rbprm \
+					robotpkg-py38-solo-rbprm \
+					robotpkg-py38-talos-rbprm \
+```
+
+## Run demo:
+
+```
+# Source you install prefix.
+source ~/devel/workspace_sl1m/install/setup.bash
+# Locate where the Solo12 specific data are.
+export SOLO3D_ENV_DIR=/home/mnaveau/Documents/cloud_laas/Shared/Solo3D/
+# Point toward you HPP installation folder
+export INSTALL_HPP_DIR=/opt/openrobots/share
```

Comment:
I suggest to avoid this INSTALL_HPP_DIR here and in the scenario and use:
```python
import anymal_rbprm
paths = [anymal_rbprm.prefix() / "anymal-rbprm" / ...
```

Idem for Talos. 
This should be available in binaries in a day or 2.

Reply:
Great I will fix this as well

Reply:
This is not working yet...
So I won't update the code until the release in solo/talos/anymal-rbprm are done.

Reply:
I'm still working on it ^^"

Reply:
I update the loading of the files using what you intended to do on the separate packages.
I assume it is fine then to merge like this.